var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -274,10 +274,11 @@ def cmake_args(self):
     #
     # export SPACK_USER_CACHE_PATH=/tmp/spack
     # export SPACK_DISABLE_LOCAL_CONFIG=true
-    def test(self):
+
+    def test_develop(self):
```

Comment:
The test method name should relate to what is tested (`N1pMDsEx1` and `N1pMdsEx1Raja`) not when (currently version `develop`).

There is a decent chance that such a test would be relevant to the next release of the software, which would necessitate a name change that would make third party test results tracking tools (e.g., CDash) lose the history.